extensions/trace-propagators/src/main/java/io/opentelemetry/extension/trace/propagation/B3Propagator.java

```diff
@@ -33,7 +33,7 @@
  * OpenTelemetry.setPropagators(
  *   DefaultContextPropagators
  *     .builder()
- *     .addTextMapPropagator(B3Propagator.getInstance())
+ *     .addTextMapPropagator(B3Propagator.getSingleHeaderInstance())
```

Comment:
I think we should still call the single-header option `.getInstance()`. In the specs, single-header is supposed to be the default, with multi-header as configurable.

Reply:
Then what about:

```
getDefault() // Returns the default instance
getMultiHeaders()
```

I think specs are written to a specific implementation in mind that there is a Builder/Factory method that without any options returns the single header version. But as with this implementation we don't have any of these patterns it is hard to reason about default.

There are other places where we can enforce "default", for example "b3" string for env vars and java properties should be the default for single header.

Reply:
I think that's fair. Maybe the default here is only important for auto-configuration, and not the programmatic ways of accessing the instances. I don't feel overly strongly...I just don't want to do something counter-spec and have to release another breaking change before 1.0. ;)

Reply:
We can always add another method call getDefault on top of the two that returns the single header